selenium_tests/pin/page_objects/pin_page.py

```diff
@@ -0,0 +1,67 @@
+"""
+This file models some of the page objects on the Manage People Search Page.  (find_user.html)
+"""
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.pin.page_objects.base_page_object import PinBasePageObject
+import abc
+
+class PinPageLocators(object):
+    # List of WebElements found on Pin Page (Locators)
+    LOGIN_TYPE_XID_RADIO_BUTTON = (By.ID, "compositeAuthenticationSourceType3")
```

Comment:
Fixed. Modified to locate the XID radio button using by CSS selector instead on attribute value text.   (By.CSS_SELECTOR, "input[value=\"XID\"]")